go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -139,6 +139,8 @@ func init() {
 	flag.BoolVar(&currentConfig.EnforceStrictTransTables, "enforce_strict_trans_tables", defaultConfig.EnforceStrictTransTables, "If true, vttablet requires MySQL to run with STRICT_TRANS_TABLES or STRICT_ALL_TABLES on. It is recommended to not turn this flag off. Otherwise MySQL may alter your supplied values before saving them to the database.")
 	flagutil.DualFormatBoolVar(&enableConsolidator, "enable_consolidator", true, "This option enables the query consolidator.")
 	flagutil.DualFormatBoolVar(&enableConsolidatorReplicas, "enable_consolidator_replicas", false, "This option enables the query consolidator only on replicas.")
+	flagutil.DualFormatInt64Var(&currentConfig.ConsolidatorStreamQuerySize, "consolidator_stream_query_size", defaultConfig.ConsolidatorStreamQuerySize, "Configure the stream consolidator query size. Setting to 0 disables the stream consolidator.")
+	flagutil.DualFormatInt64Var(&currentConfig.ConsolidatorStreamTotalSize, "consolidator_stream_total_size", defaultConfig.ConsolidatorStreamTotalSize, "Configure the stream consolidator total size. Setting to 0 disables the stream consolidator.")
```

Comment:
What is the unit for these values? We should document it in the flag help.
Also since we are introducing new flags, let us use just hyphens and not do DualFormat at all.
cc @ajm188 - do we have this guidance on dashes vs underscores written down somewhere?

Reply:
Hyphenated and documented!

Reply:
we don't, but we should. do you have a suggestion on where that should live? i'm happy to do the writing, i just don't know if in vitessio/vitess, or in enhancements, or on the website is the right place for it to live

Reply:
How about adding a section to the [contribute page](https://vitess.io/docs/15.0/contributing/code-reviews/) on the website?
And optionally, adding it to the [PR template](https://github.com/vitessio/vitess/blob/main/.github/pull_request_template.md) and/or [review template](https://github.com/vitessio/vitess-bot/blob/main/index.js#L9).
It might even be worth adding a new `CLI` section to the review template.